pkg/descheduler/strategies/topologyspreadconstraint.go

```diff
@@ -114,9 +115,10 @@ func RemovePodsViolatingTopologySpreadConstraint(
 			(len(excludedNamespaces) > 0 && excludedNamespaces.Has(namespace.Name)) {
 			continue
 		}
+		klog.V(1).InfoS("Processing namespace", "namespace", namespace.Name)
```

Comment:
Please put changes in TSC into its own PR as it's not related to the deviation feature.